src/layouts/MainLayout/providers/AppProvider.tsx

```diff
@@ -89,6 +132,66 @@ function getHTTPContext() {
 	return window.isSecureContext;
 }
 // reducers
+const setCleanAppStore = (store: AppStore) => {
```

Comment:
this reducer is synchronising the date time.  setCleanAppStore is not a good name for this reducer

Reply:
Addressed this